### ISSUE TITLE
About the organization of StanfordCars and DomainNet dataset

### ISSUE THREAD

**Issue by ShiWuxuan** [Zhanxin-Gao/CPrompt#6]
Hi, thanks for your nice work! I had some difficulty in reproducing the CPrompt. 

First, the training and test sets are sorted separately in the StanfordCars dataset downloaded from the link provided in this repository:
```
├── StanfordCars 
│   └── car_train
│       └── 00001.jpg
│       └── 00002.jpg
│       └── ......
│   └── car_test
│       └── 00001.jpg
│       └── 00002.jpg
│       └── ......
```
According to the file given in this repository (under ./data/datautils/stanfordcars), the data for the test set and the training set are in one directory, like:
```
├── StanfordCars 
│   └── car_img
│        (train)
│       └── 00001.jpg
│       └── 00002.jpg
│       └── ......
│        (test)
│       └── 00045.jpg
│       └── 00046.jpg
│       └── ......
```
**How should I divide the data?**

Additionally, the data in the site pointed to by the DomainNet link [http://ai.bu.edu/M3SDA/](http://ai.bu.edu/M3SDA/) seems to be out of date, is there another source for that dataset?

Thank you and look forward to your reply.

**Comment by Zhanxin-Gao** [comment]
Hi,
1. You only need to place all the images from both the training set and test set into the car_img directory. The provided code will handle the rest.
2. The DomainNet link still works. The dataset includes six diverse domains: Clipart, Infograph, Painting, Quickdraw, Real, and Sketch. You will need to download all of them and perform some preprocessing.

**Comment by ShiWuxuan** [comment]
Thank you for your reply, the issue about DomainNet has been resolved. 

For the problem of StanfordCars, you may have misunderstood my question. The .txt file you provided renumbers the test set and dataset images. If directly merge the training set and test set, images with duplicate numbers will be overwritten.

**Comment by Yanyi-Zhang** [comment]
Hi. I have the same problem about the StanfordCars. Did you fix it?

**Comment by Zhanxin-Gao** [comment]
Sorry, I originally downloaded the StanfordCars dataset from the official website, but when uploading the code the official website had become unavailable. Therefore I provided a Kaggle link of StanfordCars, not realizing the differences in data organization. I will directly upload the StanfordCars dataset to GitHub within the next two days.  You can also refer to the solution in https://blog.csdn.net/weixin_41735859/article/details/102986026 for reference. Thanks for your understanding.

**Comment by ShiWuxuan** [comment]
Thank you for your reply, it is very helpful to me.

**Comment by Yanyi-Zhang** [comment]
Many thanks for your help!

**Comment by Zhanxin-Gao** [comment]
Best wishes!
